app/serializers/user_base_serializer.rb

```diff
@@ -1,6 +1,6 @@
 class UserBaseSerializer < BaseSerializer
 
-  attributes :id, :email, :first_name, :is_admin, :last_name
+  attributes :id, :email, :first_name, :has_sidebar, :is_admin, :last_name
```

Comment:
We don't need to include `has_sidebar` in the `user` object, since we'll always be using the `Profile...Serializer` for current user stuff.